airflow/cli/commands/provider_command.py

```diff
@@ -189,7 +189,20 @@ def auth_backend_list(args):
         data=list(ProvidersManager().auth_backend_module_names),
         output=args.output,
         mapper=lambda x: {
-            "api_auth_backand_module": x,
+            "api_auth_backend_module": x,
+        },
+    )
+
+
+@suppress_logs_and_warning
+@providers_configuration_loaded
+def auth_managers_list(args):
+    """List all auth managers at the command line."""
+    AirflowConsole().print_as(
+        data=list(ProvidersManager().auth_managers),
```

Comment:
looks like `auth_managers` property return `list[str]`. Do we still need the `list` wrap here?

Reply:
i will fix this changes @Lee-W